pkg/platforms/openshift/openshift.go

```diff
@@ -303,12 +303,13 @@ func (c *openshiftContext) GetNodeMachinePoolName(ctx context.Context, node *cor
 		}
 	}
 
-	log.Log.Error(nil, "getNodeMachinePool(): Failed to find the MCP of the node")
-	return "", fmt.Errorf("getNodeMachinePool(): Failed to find the MCP of the node")
+	logger.Error(nil, "Failed to find the MCP of the node")
+	return "", fmt.Errorf("failed to find the MCP of the node")
```

Comment:
whether or not to log errror before return, we should be consistent one way or the other.